app/controllers/laa_references_controller.rb

```diff
@@ -23,7 +23,8 @@ def create
       if link_laa_reference
         redirect_after_link
       else
-        redirect_after_failed_link
+        flash.now[:alert] = I18n.t('laa_reference.link.failure', error_messages: error_messages)
+        render 'new'
       end
     else
       render 'new'
```

Comment:
not a blocker - its a shame e have to have two `render 'new'` - is there anyway we can get around this? 🤔

Reply:
Have used a return to achieve this, don't know if good ruby style though?